ogcapi-stable/ogcapi-features-core/src/main/java/de/ii/ogcapi/features/core/domain/FeaturesCoreQueriesHandler.java

```diff
@@ -43,7 +44,21 @@ interface QueryInputFeatures extends QueryInput {
 
     boolean getShowsFeatureSelfLink();
 
-    boolean getIncludeLinkHeader();
+    @Value.Default
+    default boolean sendResponseAsStream() {
+      return true;
+    }
+
+    // TODO rename and generalize when all file handling is moved to xtraplatform-spatial
+    @Value.Default
```

Comment:
This TODO is a reference to #877.

When a file is passed to the query handler using this option, it will not only generate the response, but it will also store the response content in the file.

Reply:
Thanks, I guess I did not make myself clear enough. My question is what are the use cases for the query handler to save a file?

Reply:
To store it in the cache.

Reply:
So would that be redundant when we integrate it with `xtraplatform-tiles`? 
I don't think a query handler should be responsible for saving files. Can't you just return the byte array and save it in the 3dtiles code? That way we wouldn't introduce unclean design into a mature module.

Reply:
Yes, it would be redundant, but I can also remove it now from Features Core and handle it in 3D Tiles. Will make the change.

Reply:
Great, thanks!